package sv_cl_messages;

message FullyConnected
{
}